de-lib/src/main/java/org/iplantc/de/theme/base/client/diskResource/details/DetailsViewDefaultAppearance.java

```diff
@@ -201,6 +203,38 @@ public String loadingMask() {
         return iplantDisplayStrings.loadingMask();
     }
 
+    @Override
+    public String searchTags() {
+        return displayStrings.searchTags();
+    }
+
+    @Override
+    public String removeTag() {
+        return displayStrings.removeTag();
+    }
+
+    @Override
+    @SuppressWarnings("unusable-by-js")
+    public Splittable tagSearchListStyle() {
+        return StringQuoter.split("{\"fontSize\": \"small\",\"width\": \"100%\"};");
+    }
+
+    @Override
+    @SuppressWarnings("unusable-by-js")
+    public Splittable infoTypeSelectStyle() {
+        return StringQuoter.split("{\"width\": \"100px\",\"fontSize\": \"small\"};");
```

Comment:
This would probably be more readable in `DetailsViewStyle.gss` - looks like some other CSS styles were added there already

Reply:
`gss` returns a class name. MUI components can't be styled with `className` like a regular `HTML` tag since they don't have that property.